examples/todo/src/Component/List.purs

```diff
@@ -1,79 +1,84 @@
-module Component.List where
+module Example.Todo.Component.List where
 
 import Prelude
 
 import Data.Array (snoc, filter, length)
 import Data.Map as M
 import Data.Maybe (Maybe(..), fromMaybe)
+import Data.Symbol (SProxy(..))
+import Example.Todo.Model (List, TaskId, initialList, initialTask)
+import Example.Todo.Component.Task (TaskQuery(..), TaskMessage(..), TaskSlot, task)
 import Halogen as H
 import Halogen.HTML as HH
 import Halogen.HTML.Events as HE
-import Model (List, TaskId, initialList, initialTask)
-import Component.Task (TaskQuery(..), TaskMessage(..), task)
 
--- | The list component query algebra.
-data ListQuery a
-  = NewTask a
-  | AllDone a
-  | HandleTaskMessage TaskId TaskMessage a
+-- | The list component query algebra (queried from the outside).
+type ListQuery a = Unit
```

Comment:
Two things: if you want to use this as a query type, it has to actually be a data type or newtype (like `type ListQuery = Const Void` or `Const Unit`), since type synonyms can't be partially applied. But since you aren't using it anywhere, maybe it is best to leave it off entirely.

Reply:
Great catch. Thank you. Will fix in next commit